src/ontology/edits/oeo-shared.omn

```diff
@@ -677,10 +680,14 @@ ObjectProperty: OEO_00010231
 
     Annotations: 
         <http://purl.obolibrary.org/obo/IAO_0000115> "A relation between an entity and an information artifact.",
-        rdfs:label "has information content entity"@en,
```

Comment:
We could leave this label as alternative label. Any opinions on that?

Reply:
I would agree if we had simply found a better name. But in this case, the old name isn't helpful. So I don't think the alternative label here would provide any additional information, as that is the case when, e.g. acronyms are given as alternative labels.
The only benefit would be for easier usage, as the old label remains somewhat in the ontology, and the property could still be found by using it. But since the `term tracker annotation` still has the old label in it, when searching for `has information content entity` in Protégé, the property is still the first result.

Reply:
ok, sounds reasonable.